src/utils/cardsApi.js

```diff
@@ -0,0 +1,24 @@
+const fetch = require('node-fetch');
+
+const baseUrl = 'http://retropolisbe-env.eba-xifiu248.eu-west-2.elasticbeanstalk.com/cards';
+
```

Comment:
We should add this url to .env variables

Reply:
Agreed, we should definitely do that!